src/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Add copyright - should be visible per file.

Reply:
Done.